### PR TITLE
Use standardpath utils to locate opencc profiles

### PR DESCRIPTION
**Pull request by WhiredPlanck**
This is useful for Android platform since it doesn't support any "standard" path of Linux.

### CODE REVIEW

**Comment by WhiredPlanck** [comment]
LGTM on PC (AMD64)

图片

**Comment by WhiredPlanck** [comment]
LGTM on Android (Emulator, x86_64)

File: modules/chttrans/chttrans-opencc.cpp [hunk 1]
```diff
@@ -5,30 +5,45 @@
...
 bool OpenCCBackend::loadOnce(const ChttransConfig &config) {
     updateConfig(config);
     return true;
 }
 
 void OpenCCBackend::updateConfig(const ChttransConfig &config) {
-    auto s2tProfile = *config.openCCS2TProfile;
-    if (s2tProfile.empty()) {
-        s2tProfile = OPENCC_DEFAULT_CONFIG_SIMP_TO_TRAD;
+    using namespace fcitx;
```
Comment (wengxt) [inline modules/chttrans/chttrans-opencc.cpp:17]: move this line to top of the file.

File: modules/chttrans/chttrans-opencc.cpp [hunk 2]
```diff
@@ -5,30 +5,45 @@
...
+    
+    auto s2tProfile = config.openCCS2TProfile->empty()
```
Comment (wengxt) [inline modules/chttrans/chttrans-opencc.cpp:19]: create a common function for this part to avoid code duplication.

**Review by wengxt — CHANGES_REQUESTED** [review]

**Comment by WhiredPlanck** [comment]
The test results above are still valid.

**Pull request merged**